fastlane/Fastfile

```diff
@@ -39,69 +54,14 @@ lane :run_api_tests do |options|
   analyze_api_tester
 end
 
-platform :ios do
-  desc "Increment build number"
-  lane :bump do |options|
-    new_version_number = options[:version]
-    fail ArgumentError, "missing version" unless new_version_number
-    version_bump_podspec(path: "ios/purchases_flutter.podspec", version_number: new_version_number)
-    version_bump_podspec(path: "macos/purchases_flutter.podspec", version_number: new_version_number)
-    replace_in("#{current_version}", "#{new_version_number}", "../ios/Classes/PurchasesFlutterPlugin.m")
-  end
-end
-
-platform :android do
-  desc "Increment build number"
-  lane :bump do |options|
-    new_version = options[:version]
-    previous_version = options[:current_version]
-
-    fail ArgumentError, "missing version" unless new_version
-    gradle_file_path = "../android/build.gradle"
-    plugin_file_path = "../android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java"
-
-    replace_in(previous_version, new_version, gradle_file_path)
-    replace_in("#{current_version}", "#{new_version}", plugin_file_path)
-  end
-end
-
-def attach_changelog_to_master
-  current_changelog = File.open("../CHANGELOG-LATEST.md", 'r')
-  master_changelog = File.open("../CHANGELOG.md", 'r')
-
-  current_changelog_data = current_changelog.read
-  master_changelog_data = master_changelog.read
-
-  current_changelog.close
-  master_changelog.close
-
-  File.open("../CHANGELOG.md", 'w') { |master_changelog_write_mode|
-    whole_file_data = "#{current_changelog_data}\n#{master_changelog_data}"
-    puts "going to save. Contents - #{whole_file_data}"
-
-    master_changelog_write_mode.write(whole_file_data)
-  }
-end
-
-def current_version
+def current_version_number
```

Comment:
Just did the rename to keep it consistent with iOS and android. Also, I haven't updated to use a `.VERSION` file but I might do that in a future PR to keep consistency in all our SDKs